lib/components/molecules/Toggle/Toggle.style.js

```diff
@@ -0,0 +1,84 @@
+import { css } from 'styled-components';
+
+const styles = css`
+
+  ${props => `
+       ${props.theme.typography.FONT_DEFAULT};
+`};
+section{
+  ${props => `
+    padding-top: ${props.theme.spacing.unit1}
+    `};
+}
+  label {
+    position: relative;
```

Comment:
@vinodloha  What did we decide for Tailwind ? Because properties like these should be part of a "position.style.js"